Paralayout/UIView+Alignment.swift

```diff
@@ -229,19 +229,35 @@ extension UIView {
             offset: .init(horizontal: superviewPoint.x + horizontalOffset, vertical: superviewPoint.x + verticalOffset)
         )
     }
-    
-    /// Move the view to align it with another view.
+
+    /// Move the view to align it within its superview.
     ///
     /// - precondition: The receiver must have a superview.
     ///
     /// - parameter position: The position in both the receiving view and its `superview` to use for alignment.
-    /// - parameter inset: An optional inset (horizontal, vertical, or diagonal based on the position) to apply. An
-    /// inset on .center is interpreted as a vertical offset.
-    public func alignToSuperview(_ position: Position, inset: CGFloat = 0.0) {
+    /// - parameter horizontalOffset: An additional horizontal offset to apply to the receiver. Defaults to no offset.
+    /// - parameter verticalOffset: An additional vertical offset to apply to the receiver. Defaults to no offset.
+    public func alignToSuperview(_ position: Position, horizontalOffset: CGFloat = 0, verticalOffset: CGFloat = 0) {
```

Comment:
Now, if we call `.alignToSuperview(.center)` (with assumed default parameters), this functions will be called, not the 
```
public func alignToSuperview(_ position: Position, inset: CGFloat)
```
as before. Does 
```
public func alignToSuperview(_ position: Position, horizontalOffset: CGFloat = 0, verticalOffset: CGFloat = 0)
```
behave the same?

Reply:
Yep, exactly. I removed the default value for `inset` so that we could set defaults for `horizontalOffset` and `verticalOffset` without `alignToSuperview(.center)` become ambiguous. An inset of zero (the old default) and an offset of zero in both axes should result in exactly the same behavior.